automated install/basic-install.sh

```diff
@@ -268,7 +268,7 @@ package_manager_detect() {
         # These variable names match the ones for apt-get. See above for an explanation of what they are for.
         PKG_INSTALL="${PKG_MANAGER} install -y"
         # CentOS package manager returns 100 when there are packages to update so we need to || true to prevent the script from exiting.
-        PKG_COUNT="${PKG_MANAGER} check-update | grep -E '(.i686|.x86|.noarch|.arm|.src|.riscv64)' | wc -l || true"
+        PKG_COUNT="${PKG_MANAGER} check-update --quiet | grep -E '(.i686|.x86|.noarch|.arm|.src|.riscv64)' | wc -l || true"
```

Comment:
That's not what I meant when I commented on not redirecting to /dev/null. My point was to not hide any output, no quiet mode and no redirection.